pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.8

```diff
@@ -10,7 +10,7 @@
 .Sh SYNOPSIS
 .Nm
 .Bro
-.Cm switch | boot | test | build | dry-build | dry-activate | edit | build-vm | build-vm-with-bootloader
+.Cm switch | boot | test | build | dry-build | dry-activate | edit | build-vm | build-vm-with-bootloader | list-generations Op Fl -json
```

Comment:
I'm not experienced at all with manpages writing, but for sure this shouldn't be this:

```suggestion
.Cm switch | boot | test | build | dry-build | dry-activate | edit | build-vm | build-vm-with-bootloader | list-generations Op Fl --json
```

?

Reply:
I think the second "-" is taken care of by the `Fl`. At least it is shown properly as `--json` when I run `man pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.8`. I have to say though, that I am totally unfamiliar with this man-page syntax...

Reply:
> At least it is shown properly as `--json` when I run `man pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.8`.

I trust you on that :+1:.

Reply:
You don't have to: After having built the whole thing many times from scratch I read somewhere that you can just run `man <path to file.8>`. Makes life a lot easier when dealing with this "interesting" syntax 😂 

Actually now that I think about it: my very first version of this (at least the one that was not the old XML doc) also had the same code you suggested, but that showed `---json` and I fixed it later ^^